app/tests/unit/routes/v1/admin.spec.js

```diff
@@ -0,0 +1,144 @@
+const request = require('supertest');
+const Problem = require('api-problem');
+
+const helper = require('../../../common/helper');
+
+//
+// mock middleware
+//
+const keycloak = require('../../../../src/components/keycloak');
+//
+// test assumes that caller has appropriate token, we are not testing middleware here...
+//
+keycloak.protect = jest.fn(() => {
+  return jest.fn((req, res, next) => {
+    next();
+  });
+});
```

Comment:
This appears pretty frequently in all the new unit test files. As this is mocking out keycloak.protect, ideally this should be wrapped into a beforeAll or beforeEach clause, and then a `keycloak.protect.mockRestore()` in an afterEach or afterAll clause so that there's a stronger guarantee that these unit tests don't accidentally end up trampling on other unrelated unit tests that also happen to mock out keycloak.protect in some different capacity.

Same advice applies for the other two mocks below (userAccess, service).